resources/js/components/ui/form/Checkbox.tsx

```diff
@@ -0,0 +1,58 @@
+import {Component, JSX, createEffect, splitProps} from "solid-js";
+import * as checkbox from "@zag-js/checkbox";
+import {normalizeProps, useMachine} from "@zag-js/solid";
+import {createMemo, createUniqueId} from "solid-js";
+import s from "./Checkbox.module.scss";
+import {ValidationMessages} from "./ValidationMessages";
+import {useFieldLabel} from "./FieldLabel";
+
+export interface CheckboxProps extends Omit<JSX.InputHTMLAttributes<HTMLInputElement>, "onChange"> {
+  name: string;
+  label?: string;
+  disabled?: boolean;
+  onChange?: checkbox.Context["onChange"];
```

Comment:
To chyba powinnno również wylecieć (tak jak checked o którym piszę niżej ale wcześniej). Jak ktoś chce reagować na zmiany to powinien sobie zrobić effect na odpowiednią część felte forma, zgodnie z tym co mówiliśmy na discordzie.